projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -333,7 +340,10 @@ export class AuthService implements OnDestroy {
         if (!isLoading) {
           this.refreshState$.next();
         }
-        const target = result?.appState?.target ?? '/';
+        const appState = result.appState;
```

Comment:
Any reason why we changed `result?.appState` to be `result.appState`?

Reply:
Pardon me, this was just a simple oversight on my part